pkg/apis/app/v1alpha1/types.go

```diff
@@ -35,6 +35,7 @@ type FlinkApplicationSpec struct {
 	FlinkVersion      string                       `json:"flinkVersion"`
 	TaskManagerConfig TaskManagerConfig            `json:"taskManagerConfig,omitempty"`
 	JobManagerConfig  JobManagerConfig             `json:"jobManagerConfig,omitempty"`
+	IngressConfig     IngressConfig                `json:"ingressConfig,omitempty"`
```

Comment:
@annymsMthd 

Right now for an application when an Ingress is created, it is never updated (unlike Service and Deployment). Let's say you add an annotation for Ingress config, and you update it, the update will never be reflected in the Ingress.

Reply:
Okay. I can add that if you guys like this addition to the api.

Reply:
Will have to think it through. We currently do not have this requirement.

We pass on the annotation from the flink application to the deployments. Would doing something similar for ingress and services address your use case?

Reply:
That would work.

Reply:
Sorry for the delay. Forgot about this thread.

Submitted a PR: https://github.com/lyft/flinkk8soperator/pull/74